docs/v0.9/en/user-guide/concepts/features-that-you-concern.md

```diff
@@ -1,5 +1,11 @@
 # Features that You Concern
 
+## Does GreptimeDB support logs or events?
+
+Yes, since v0.9.0, GreptimeDB treats all time series as contextual events with timestamps, and thus unifies the processing of metrics, logs, and events. It supports analyzing metrics and events with SQL, PromQL, and streaming with continuous aggregation.
```

Comment:
```suggestion
Yes. Since v0.9.0, GreptimeDB treats all time series as contextual events with timestamps, and thus unifies the processing of metrics, logs, and events. It supports analyzing metrics, logs, and events with SQL, PromQL, and streaming with continuous aggregation.
```